public/components/common/tables/table-with-search-bar.tsx

```diff
@@ -79,31 +81,46 @@ export function TableWithSearchBar({
     }
   }, [endpoint, reload]);
 
-  useEffect(function () {
-    (async () => {
-      try {
-        setLoading(true);
-        const { items, totalItems } = await onSearch(endpoint, filters, pagination, sorting);
-        setItems(items);
-        setTotalItems(totalItems);
-      } catch (error) {
-        setItems([]);
-        setTotalItems(0);
-        const options = {
-          context: `${TableWithSearchBar.name}.useEffect`,
-          level: UI_LOGGER_LEVELS.ERROR,
-          severity: UI_ERROR_SEVERITIES.BUSINESS,
-          error: {
-            error: error,
-            message: error.message || error,
-            title: `${error.name}: Error fetching items`,
-          },
-        };
-        getErrorOrchestrator().handleError(options);
-      }
-      setLoading(false);
-    })();
-  }, [filters, pagination, sorting]);
+  useEffect(
+    function () {
+      (async () => {
+        try {
+          setLoading(true);
+          let { items, totalItems } = await onSearch(
+            endpoint,
+            filters,
+            pagination,
+            sorting,
+          );
+          if (compressipv6) {
+            items.forEach(item => {
+              if (item.ip) {
+                item.ip = compressIPv6(item.ip);
+              }
+            });
```

Comment:
**nitpick**: This seems to be too specific to add a property to replace the `ip` property of the response to be included in the reusable component. The component has a way to interact with the response item data, this is the `mapResponseItem` component property.

**suggestion**: You have 2 options:
1. if you want to replace the value for a property of the response, you can use the `mapResponseItem` property method of the TableWzAPI component, which transforms the response item.
```tsx
<TableWzAPI
  ...
  mapResponseItem={(item) => ({
    ...item,
    ip: compressIPv6(item.ip)
  })
}
/>
```

2. if you want to keep the original value (for some reason), you can use the `render` of the table column to render what you want.

I think option 1 is the best one for this case.